### PR TITLE
#1115 sticky header course view

### LINKED ISSUE #1115: Sticky Header Course-View
# User Story

### As a: 
User / Teacher

### I want:
To have a sticky header which shows the current course and current lection and so on. 
According to Breadcrumbs. Sticky Header Ex.: 

Courses > Course XY > Lection Z > Unit Z

When clicking on each Part the page scrolls to the top of the clicked section. (Or go back to Courses View)

### PR DESCRIPTION
**Pull request by Gargamil**
## Description:
Adds a sticky header to course (teacher & student)

Closes #1115 

## Improvements
-

### CODE REVIEW

**Comment by kesselb** [comment]
@Gargamil is this ready for review?

**Comment by kesselb** [comment]
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Translatable SnackBarService. [#922](https://github.com/geli-lms/geli/issues/922)
+- Sticky header course view. [#1115](https://github.com/geli-lms/geli/issues/1115)
```
Comment (kesselb) [inline CHANGELOG.md]:
```suggestion
- Sticky header for course view. [#1115](https://github.com/geli-lms/geli/issues/1115)
```

File: app/webFrontend/src/app/shared/components/sticky-toolbar/sticky-toolbar.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (kesselb) [inline app/webFrontend/src/app/shared/components/sticky-toolbar/sticky-toolbar.component.spec.ts]: Is this test working? If not please delete it.

**Comment by Gargamil** [comment]
Fixed https://github.com/geli-lms/geli/pull/1179#issuecomment-475876091.

**Comment by kesselb** [comment]
> Intro... > Codi... > Codek....

Truncating is bad from UX perspective :thinking: You'll figure something out :+1:

**Comment by Gargamil** [comment]
Set to `overflow-x: scroll` and added some style. Should now be responsive useful as well.

top_scroll

**Comment by kesselb** [comment]
There are some conflicts with your other pr ;)

**Comment by Gargamil** [comment]
> There are some conflicts with your other pr ;)

thanks 👍  
should be resolved now.

**Pull request merged**